IPython/extensions/tests/test_autoreload.py

```diff
@@ -132,7 +132,6 @@ class TestAutoreload(Fixture):
 
     @skipif(sys.version_info < (3, 6))
     def test_reload_enums(self):
-        import enum
```

Comment:
Do you think it should be restored? This test is already skipped on Python < 3.6, so we should always be able to import enum. Might be nicest to have a clear error if that goes wrong for some reason, though.